---
layout: post
title: "Lab note 2026-01-07"
date: 2026-01-07
---

# Literature Survey 

**Date:** 2026-1-7  
**Type:** Literature survey  
**Project:**  The role of RNA binding protein in OET  
**Status:** Planned  
**Updated Date:** 2026-1-13


## Goal
The purpose is to collect functional information about Mkrn1, Mkrn2, Ptbp2 gene and their expression in Zebrafish oocyte and other tissue.

* TOC
{:toc}


# Summary
![Workflow](/assets/img/lab-notes/2026-01-07/screening_workflow_20260114.jpg)



| ID. | RBP |Phenotype | Reference |
|:-----|:------|:------------|:----------|
| 1 |**Hnrnpm** | | |
| 2 | **<font color=red>Fubp2/khsrp</font>** | Immune function; Decreased susceptibility to Herpesvirales infection induced morbidity/mortality;；related to prostate cancer | [MGI-fubp2 & Cai et al. 2024](#fubp2khsrp)|
| 3 |**<font color=grey>Fubp1</font>** | Embryonic lethality |[Zhou et al. 2016 & MGI-fubp1](#fubp1) |
| 4 | **si:dkeyp-121d4.3** | Premature Stop | [ZFIN-si:dkeyp-121d4.3](#sidkeyp-121d43)|
| 5 |**Zfr** | homozygous: embryonic lethality between somite formation and embryo turning, complete penetrance|[MGI-zfr](#zfr)|
| 6 | **Rbm26** | mRNA stability; chromatin remodeling; cell viability; preweaning lethality, incomplete penetrance | [Chatr-aryamontri et al. 2025 & Xin et al. 2023 & MGI-rbm26](#rbm26)|
| 7 |**Unk** | preweaning lethality, incomplete penetrance| [MGI-unk](#unk)|
| 8 |**Celf1** | postnatal lethality, incomplete penetrance| [MGI-celf1](#celf1)|
| 9 | **Rbm10** | proliferation defects, differentiation potential; cardiac development and cancer(lung adenocarcinoma, colorectal cancer, pancreatic cancer and bladder cancer) |[Rodor et al. 2017 & Gebauer et al. 2021](#rbm10)|
| 10 | **<font color=red>Rbm5</font>** | male infertility | [ XO'Bryan et al. 2013 &  Rodor et al. 2017](#rbm5)|
| 11 |**Bicc1a** | preweaning lethality, incomplete penetrance |[MGI-bicc1](#bicc1a) |
| 12 | **<font color=grey>Hnrnpk</font>** | axonogenesis of embryo; reduced survival |[Geuens et al. 2016](#hnrnpk)|
| 13 | **Fubp3** | Proto-oncogene; Viral replication | [Izsvák et al. 2010](#fubp3)|
| 14 | **<font color=grey>Rbm15</font>** | embryonic lethality | [Hu et al. 2020 & MGI-rbm15](#rbm15)|
| 15 | **Rbm28** | preweaning lethality, complete penetrance |[ Lin et al. 2022 & MGI-rbm28](#rbm28)|
| 16 | **Rbm39b** | prenatal lethality, complete penetrance | [MGI-rbm39b](#rbm39b)|
| 17| **Rbm19** | failure of morula compaction; embryonic growth arrest; embryonic lethality before implantation, complete penetrance | [MGI-rbm19](#rbm19)|
| 18 | **Raver2** | behavior/neurological: decreased grip strength; homeostasis/metabolism: decreased circulating phosphate level |[ MGI-raver2](#raver2)|
| 19 | **Puf60a** | tumor |[ Zhang et al. 2022](#puf60a) |
| 20 | **<font color=grey>Ptbp2a</font>** | Pan-neuronal Ptbp2 knockout mice (Ptbp2loxp/loxp; Nes-Cre) die at birth |[Vuong et al. 2016 & Zagore et al. 2015](#ptbp2a) |
| 21 | **<font color=red>Mkrn1</font>** | axis specification, pole plasm assembly | [Dold et al. 2020 & Gray et al. 2006](#mkrn1)|
| 22 | **<font color=red>Mkrn2</font>** | male infertility, abnormal spermiogenesis | [ Wang et al. 2023 & Qian et al. 2016 & MGI-mkrn2](#mkrn2)|
| 23 | **Pcbp2** |  homozygous: lethality throughout fetal growth and development, complete penetrance; postnatal lethality, complete penetrance; embryonic lethality during organogenesis, incomplete penetrance | [ MGI-pcbp2](#pcbp2)|

Selection criteria: knockout of the gene does not cause embryonic lethality, and the phenotype is related to the reproductive system. Final list for antibody: fubp2, rbm5, mkrn1, mkrn2.

**Crispr design:**
- fubp2: [3 transcripts](https://useast.ensembl.org/Danio_rerio/Transcript/Summary?db=core;g=ENSDARG00000026489;r=1:54960189-54972236;t=ENSDART00000150548), 5 common exons
  ```bash
  >khsrp-201_ENSDARE00000296842_exon:protein_coding, khsrp-202_ENSDARE00000296842_exon:protein_coding, khsrp-203_ENSDARE00000296842_exon:protein_coding
  ATTGCAGCAAAGATCGGTGGTGACGTTACAGGTCCCCCAATGAGCAATAACGGCGGAGCGGAGAGTTATCCCTTCCCGACGCCGAAGCGATCGCTGGAGGATGGAG
  >khsrp-201_ENSDARE00000787950_exon:protein_coding, khsrp-202_ENSDARE00000787950_exon:protein_coding, khsrp-203_ENSDARE00000787950_exon:protein_coding
  CTATCGGAGCCCAGCTGGCTGCCCTTTCCAAGCAGAG
  >khsrp-201_ENSDARE00000238850_exon:protein_coding, khsrp-202_ENSDARE00000238850_exon:protein_coding, khsrp-203_ENSDARE00000238850_exon:protein_coding
  TGTTCGTCCTTCGTCGATGACAGAAGAGTACAGAGTCCCTGATGGCATGGTTGGCCTCA
  >khsrp-201_ENSDARE00000292534_exon:protein_coding, khsrp-202_ENSDARE00000292534_exon:protein_coding, khsrp-203_ENSDARE00000292534_exon:protein_coding
  TCATTGGTCGAGGTGGCGAGCAGATCAATAAGATCCAGCAGGACTCTGGTTGTAAAGTTCAGATCGCTCCAG
  >khsrp-201_ENSDARE00001227839_exon:protein_coding, khsrp-202_ENSDARE00001227839_exon:protein_coding, khsrp-203_ENSDARE00001227839_exon:protein_coding
  ACAGTGGCGGTCTTCCAGACAGAAGTGTGTCCATCACAGGAGGCCCTGAAGCCATCCA
  ```
  [Chopchop: fubp2(ENSDARG00000026489)](https://chopchop.cbu.uib.no/results/282c10f0-d547-477b-93da-12b1b363189c/)

  **khsrp-201_ENSDARE00000296842:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=T9egZ5S8AsR7IHUBcRv4), [RGEN](http://www.rgenome.net/cas-designer/result?hash=64732fff31a9dfa1a5782aa4e2ff0f8a)
  
  **khsrp-201_ENSDARE00000787950:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=DPAihT8wXnSC2aLDPdSC), [RGEN](http://www.rgenome.net/cas-designer/result?hash=87fb0d7011f0ba8f706dfbbfb3aee907)

  **khsrp-201_ENSDARE00000238850:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=OS7EQmbebs8HKmdK81iL),[RGEN](http://www.rgenome.net/cas-designer/result?hash=68d490bcbd31c53937bd2b0a0afc8d55)

  **khsrp-201_ENSDARE00000292534:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=VKbWKBBx0JP6azGcxaDK),[RGEN](http://www.rgenome.net/cas-designer/result?hash=95100dc71c6f7fdd49171eafd7b61796)

  **khsrp-201_ENSDARE00001227839:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=nKAe2WFrEnxKJzU4WkY8), [RGEN](http://www.rgenome.net/cas-designer/result?hash=8fdcfa685ac7ad6fb6d0f9fdf1becac9)  

  **Candidate1:** TACTCTTCTGTCATCGACGAAGG
  
  **Candidate2:** GATCGCTTCGGCGTCGGGAAGGG




- rbm5: [4 transcripts](https://may2025.archive.ensembl.org/Danio_rerio/Gene/Summary?db=core;g=ENSDARG00000098280;r=6:53263854-53282595), 2 common exons
  ```
  >RBM5-201_ENSDARE00001157869_exon:nonsense_mediated_decay
  GAGGAGATCCAGCCGATGAAAAGGAGAGCAAGACCATCATGCTCCGGGGTCTGCCCATGAACACGTCAGAGGCGGAT
  >RBM5-201_ENSDARE00001206436_exon:nonsense_mediated_decay, RBM5-204_ENSDARE00001206436_exon:protein_coding, RBM5-202_ENSDARE00001206436_exon:protein_coding
  ATTCGAGCAGCCATTGACTTGCTGGAAGGACCCAAGCCAATGGACGTGAGGCTGATGAAAAAGAGAACAG
  ```
  [Chopchop: rbm5(ENSDARG00000098280)](https://chopchop.cbu.uib.no/results/526284d4-456c-4ab5-83d8-becf07629eba/)

  **RBM5-201_ENSDARE00001157869:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=XmABQZh3Qu6OLdxgCdUN), [RGEN](http://www.rgenome.net/cas-designer/result?hash=548f77d2c138074d8d6509bab53a25b2)

  **RBM5-201_ENSDARE00001206436:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=H4lnkkKEqssrYotbFJb6), [RGEN](http://www.rgenome.net/cas-designer/result?hash=0c5c30050ba0e24b6da065fa99b76fb1)

  **Candiadte1:** ACGTGTTCATGGGCAGACCCCGG  

  **Candiadte2:** GACCCAAGCCAATGGACGTGAGG  

- mkrn1: [3 transcripts](https://may2025.archive.ensembl.org/Danio_rerio/Gene/Summary?db=core;g=ENSDARG00000041665;r=4:12292274-12307834;t=ENSDART00000133672), 1 common exon
  ```
  >mkrn1-207_ENSDARE00001064427_exon:protein_coding
  GCGGCAGCGTCAACAGCGGCTCCAGCAGTTATCGGAGGCTGGACGAAGCACGTAACCTGCAG
  ```
  [Chopchop: mkrn1(ENSDARG00000041665)](https://chopchop.cbu.uib.no/results/af48af78-01e6-4e48-9438-4faa66f31128/)
  **mkrn1-207_ENSDARE00001064427:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=8mNc1LPm1J2q8DIEpT6e), [RGEN](http://www.rgenome.net/cas-designer/result?hash=391ed6c3672af210a35501eb846592d8)
  **Candidate1:** GCGGCTCCAGCAGTTATCGGAGG

- mkrn2: [3 transcripts](https://may2025.archive.ensembl.org/Danio_rerio/Gene/Summary?db=core;g=ENSDARG00000007630;r=11:543365-575786), 6 common exons
  ```
  >MKRN2-203_ENSDARE00000500305_exon:protein_coding, MKRN2-201_ENSDARE00000500305_exon:protein_coding, MKRN2-204_ENSDARE00000500305_exon:protein_coding
  GTATTTCCTCCACGGTGTGTGTCGGGAGGGCAGCAGGTGTCTGTTTTCTCACGATTTAACCACAAGTAAACCTTCCACCATCTGCAAATACTACCAGAGAGGAGCGTGTGCTTATGGAGACAGATGCAG
  >MKRN2-203_ENSDARE00000059816_exon:protein_coding, MKRN2-201_ENSDARE00000059816_exon:protein_coding, MKRN2-204_ENSDARE00000059816_exon:protein_coding
  ATATGACCATATTAAACCTCCTGGTCGTGGCAGTGGAGCTCCAGCAGATCACAGTAACAGAAGCAGCAGCAGTGCAGGAGCCTCCGCACCCGGCCCCGGACCCCCGGCAAACACCAGCAAACACCTCAAAAAACCACTAGTGCTTCGAGACAAAG
  >MKRN2-203_ENSDARE00000842502_exon:protein_coding, MKRN2-201_ENSDARE00000842502_exon:protein_coding, MKRN2-204_ENSDARE00000842502_exon:protein_coding
  CTCTGTGCTCAGACAGCAGACCTCGTGTGTTCAGTGCGGAGAGTTCAGAGCTCAGTGAATGCTGGGAGCAGCGAGACGACGGCGCTCAGAAGCCGCACTCGTATCTGGAGGCCATCCGCTCTGGCCTGGACGCCTCCGCCGCCGCCGCTGCCACCGCTGGGACGTTTCCAGAGCTCCAGCAGACGTCGCCGCAGATCTGCCCGTTCCTGGCCGCAGGGCAGTGCCAGTATGGAGAAAGCTGTCCGTATCTGCATGGGGAGATGTGTGAGATCTGCAGACAGCATGTGCTGCATCCACACGACCCCGAGCAGAGAGCAGCGCACGAGAAG
  >MKRN2-203_ENSDARE00000500304_exon:protein_coding, MKRN2-201_ENSDARE00000500304_exon:protein_coding, MKRN2-204_ENSDARE00000500304_exon:protein_coding
  AAGTGTATGGTGGCGTTTGAGATGGACATGGAGCGAGCGTTCGCAGTGCAGCAGAGTCAGGATAAGGTGTGTAAGATCTGTCTGGATGTGGTGTACGAGAAATCCTCTCCGTCTGAGCGGCGCTTCGGCATCCTGTCCAGCTGCGCACACACATACTGCCTGAACTGCATCCGCCAGTGGCGCTGCGTAGAGCAGTTACACAACCAGATACGCAA
  >MKRN2-203_ENSDARE00000147116_exon:protein_coding, MKRN2-201_ENSDARE00000147116_exon:protein_coding, MKRN2-204_ENSDARE00000147116_exon:protein_coding
  GTCTTGTCCAGAGTGTCGTGTGGTCTCAGAGTTCGTGATTCCCAGCATCTACTGGGTGGAGGATCAGGAGCAGAAAAACCTGCTCATCGAGGAGTTCAAGTCTGGAGTCAG
  >MKRN2-203_ENSDARE00000143695_exon:protein_coding, MKRN2-201_ENSDARE00000143695_exon:protein_coding, MKRN2-204_ENSDARE00000143695_exon:protein_coding
  TAAGAAAGCCTGTAAGTACTTTGACCAGGGCAGAGGAACGTGTCCGTTTGGAGGGAAGTGTTTCTACATGCACGCTTATGCTGACGGCAGACGAGCAGAACCTGACAAACCCCGAAAACAGCTCAGCGCTGAGGGAAACGTGCGG
  ```

  [Chopchop: mkrn2(ENSDARG00000007630)](https://chopchop.cbu.uib.no/results/fb6781f8-2943-4ad1-b5d6-bb2e34eb665f/)
  **MKRN2-203_ENSDARE00000500305:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=iMg69SKtbpOPihOhq89b),[RGEN](http://www.rgenome.net/cas-designer/result?hash=5bba9afb5b7606573ece2e48bada6aea)  

  **MKRN2-203_ENSDARE00000059816:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=nJRew39dNVoisS4kLxp8),[RGEN](http://www.rgenome.net/cas-designer/result?hash=02a8368c73eb85199d5812b668f227b0) 

  **MKRN2-203_ENSDARE00000842502:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=DhZHio3LGS6PklKXfHos),[RGEN](http://www.rgenome.net/cas-designer/result?hash=41bfe26af7919f40b9be134714ce5fd5) 

  **MKRN2-203_ENSDARE00000500304:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=0ERv57LTv4MCpZGWH6nl),[RGEN](http://www.rgenome.net/cas-designer/result?hash=86647c6131ee54e3069ac2a653604ea3) 

  **MKRN2-203_ENSDARE00000147116:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=QkDrVZa2Y40EMLySeRLS),[RGEN](http://www.rgenome.net/cas-designer/result?hash=dec2f29e90d378a9fe4c4adf06571639) 

  **MKRN2-203_ENSDARE00000143695:** [CRISPOR](https://crispor.gi.ucsc.edu/crispor.py?batchId=2Qg2QqnrKglPf5eS21aK),[RGEN](http://www.rgenome.net/cas-designer/result?hash=f16decf6adefbff17729048503ad5d51) 

  **Candidate:** GAAGCCGCACTCGTATCTGGAGG




# Antibody information
- **MKRN1**: [Makorin-1(C-8): sc-515815, SANRA CRUZ BIOTECHNOLOGY, Mouse monoclonal antibody](/assets/img/lab-notes/2026-01-07/MKRN1.jpg)
- **MKRN2**: [Makorin-2(C-5): sc-514815, SANRA CRUZ BIOTECHNOLOGY, Mouse monoclonal antibody](/assets/img/lab-notes/2026-01-07/MKRN2.jpg)
- **FUBP2**: [RQ7979, NSJ BIOREAGENTS, Rabbit IgG](/assets/img/lab-notes/2026-01-07/FUBP2.jpg)
- **RBM5**: [RMB5(E8G8K) Rabbit monoclonal antibody, Cell Signaling](/assets/img/lab-notes/2026-01-07/RBM5.jpg)
  

# Issues & Notes
Unexpected results, errors, thoughts.

## Next Actions
- start from section, immuno staining then move to whole mount


# MKRN Family

![MKRN Protein Family](https://media.springernature.com/full/springer-static/image/art%3A10.1134%2FS0006297924090037/MediaObjects/10541_2024_2739_Fig1_HTML.jpg?as=webp)


- **Paper0:** [Böhne, Astrid, et al. "The vertebrate makorin ubiquitin ligase gene family has been shaped by large-scale duplication and retroposition from an ancestral gonad-specific, maternal-effect gene." BMC genomics 11.1 (2010): 721.](https://link.springer.com/article/10.1186/1471-2164-11-721)    
  **Species:** Xiphophorus maculatus, medaka, zebrafish and amphibians   
  **Mechanism:**  Via partial sequencing of a testis cDNA library from the poeciliid fish Xiphophorus maculatus, we have identified a new member of the makorin gene family, that we called mkrn4. In addition to the already described mkrn1 and mkrn2, mkrn4 is the third example of a makorin gene present in both tetrapods and ray-finned fish. However, this gene was not detected in mouse and rat, suggesting its loss in the lineage leading to rodent murids. Mkrn2 and mkrn4 are located in large ancient duplicated regions in tetrapod and fish genomes, suggesting the possible involvement of ancestral vertebrate-specific genome duplication in the formation of these genes. Intriguingly, many mkrn1 and mkrn2 intronless retrocopies have been detected in mammals but not in other vertebrates, most of them corresponding to pseudogenes. The nature and number of zinc fingers were found to be conserved in Mkrn1 and Mkrn2 but much more variable in Mkrn4, with lineage-specific differences. RT-qPCR analysis demonstrated a highly gonad-biased expression pattern for makorin genes in medaka and zebrafish (ray-finned fishes) and amphibians, but a strong relaxation of this specificity in birds and mammals. All three mkrn genes were maternally expressed before zygotic genome activation in both medaka and zebrafish early embryos.  
  **Results:** gonad-specific, maternal-effect

## Mkrn1
- **Paper1:** [Dold, Annabelle, et al. "Makorin 1 controls embryonic patterning by alleviating Bruno1-mediated repression of oskar translation." PLoS genetics 16.1 (2020): e1008581. ](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008581)  
  **Species:** Drosophila  
  **Mechanism:** Mkrn1 interacts with poly(A) binding protein (pAbp) and binds specifically to osk 3’ UTR in a region adjacent to A-rich sequences. Mkrn1 controls embryonic patterning and germ cell formation by specifically activating osk translation, most likely by competing with Bru1 to bind to osk 3’ UTR.  
  **Results:** axis specification, pole plasm assembly 

- **Paper2:** [Gray, Todd A., et al. "The putatively functional Mkrn1-p1 pseudogene is neither expressed nor imprinted, nor does it regulate its source gene in trans." Proceedings of the National Academy of Sciences 103.32 (2006): 12039-12044.](https://www.pnas.org/doi/abs/10.1073/pnas.0602216103)  
  **Species:** Mice  
  **Mechanism:** The pseudogene of Mkrn1, called Mkrn-p1, regulates stability of the Mkrn1 mRNA in trans. Transgenic mice with reduced transcription of Mkrn1-p1 manifested polycystic kidneys and bone deformity. However, it was shown later that Mkrn1-p1 is fully methylated on both alleles; it is not transcribed and, therefore, cannot stabilize Mkrn1 transcripts in trans. Moreover, none of the phenotypes caused by a partial reduction of Mkrn1 in mutant mice were seen in the mutants with direct Mkrn1 disruption. 
  **Results:** no phenotypes of polycystic kidneys and bone deformity


## Mkrn2
- **Paper1:** [Wang, Lin, et al. "MKRN2 knockout causes male infertility through decreasing STAT1, SIX4, and TNC expression." Frontiers in Endocrinology 14 (2023): 1138096](https://www.frontiersin.org/journals/endocrinology/articles/10.3389/fendo.2023.1138096/full)  
  **Species:** Mice  
  **Mechanism:** MKRN2 regulated the expression level of SIX4 and tenascin C (TNC) via the EBF transcription factor 2 (EBF2) in mice.  
  **Results:** male infertility, abnormal function of gonad  

- **Paper2:** [Qian, Xu, et al. "Deficiency of Mkrn2 causes abnormal spermiogenesis and spermiation, and impairs male fertility." Scientific reports 6.1 (2016): 39318.](https://www.nature.com/articles/srep39318)  
  **Species:** Mice  
  **Mechanism:** Mkrn2 knockout mice produced abnormal sperms characterized by low number, poor motility, and aberrant morphology. To understand the molecular mechanism, we found that expression of Odf2, a vital protein in spermatogenesis, was significantly decreased. In addition, we found that expression levels of Odf2 were decreased in Mkrn2 knockout mice.
  **Results:** male infertility, abnormal spermiogenesis

- **Database:** [MGI-mkrn2](https://www.informatics.jax.org/allele/MGI:6163817)
  **Species:**  Mice  
  **Phenotypes:** cellular, endocrine/exocrine, growth/size/body, reproductive


## Ptbp2a
- **Paper1:** [Vuong, John K., et al. "PTBP1 and PTBP2 serve both specific and redundant functions in neuronal pre-mRNA splicing." Cell reports 17.10 (2016): 2766-2775.](https://www.cell.com/cell-reports/fulltext/S2211-1247(16)31592-3)  
  **Species:** Mice  
  **Mechanism:** Pan-neuronal Ptbp2 knockout mice (Ptbp2loxp/loxp; Nes-Cre) die at birth. PTBP1 partially rescues PTBP2 loss during brain development. PTBP1 and PTBP2 regulate overlapping but distinct sets of alternative exons in vivo. Exons exhibit developmental regulation following their sensitivity to PTBP1/2, Differential targeting is likely determined by cofactors rather than RNA binding.  
  **Results:** brain development  

- **Paper2:** [Zagore, Leah L., et al. "RNA binding protein Ptbp2 is essential for male germ cell development." Molecular and cellular biology 35.23 (2015): 4030-4042.](https://www.tandfonline.com/doi/abs/10.1128/MCB.00676-15)
  **Species:** Mice  
  **Mechanism:** Ptbp2 ablation results in germ cell loss due to increased apoptosis of meiotic spermatocytes and postmeiotic arrest of spermatid differentiation. Furthermore, we show that Ptbp2 is required for alternative splicing regulation in the testis, as in brain. Strikingly, not all of the alternatively spliced RNAs examined were sensitive to Ptbp2 loss in both tissues.  
  **Results:** spermatogenesis



# RBM Family
## Rbm26
- **Paper1:** [Chatr-aryamontri, Andrew, et al. "A platform for the systematic interrogation of genetic interactions in human cells: 26 gene by genome-wide double knockout screens as a proof of concept." bioRxiv (2025): 2025-08.](https://www.biorxiv.org/content/10.1101/2025.08.04.665369.abstract)  
  **Species:** Human cell line  
  **Mechanism:** RBM26 encodes a protein containing an RNA binding motif that, in association with Poly(A) Tail eXosome Targeting (PAXT), participates in the decay of nuclear mRNA (30) and in mRNA splicing via the recruitment of PABPN1 (31). The RBM26 gene was selected for screening because we previously identified it as a top hit in a chemogenomic screen with etoposide, a Topoisomerase 2 inhibitor (unpublished data). Although the specific function(s) of RBM26 in the DNA damage response pathway remain unclear, we reasoned that identifying its genetic interactors could be informative of its cellular roles. The most pronounced synthetic lethal hit was the cAMP-dependent Protein Kinase A gene (PKA). While one report indicates that PKA can phosphorylate XRCC6 (32), the relationship between PKA and DNA damage is not well understood. The second strongest synthetic lethal hit in the RBM26 knockout screen was the E3 ubiquitin ligase FBXW7, and the most significant rescue was the DNA ligase LIG4 (Fig. 3E). LIG4 is a component of the DNA-PK complex kinase, which includes PRKDC, XRCC4/5/6, DCLRE1C, and NHEJ1, and is involved in mediating Non-Homologous End-Joining (NHEJ) at DNA ends. FBXW7 is known to polyubiquitinate XRCC4, thereby enhancing its interaction with XRCC5 and XRCC6 (33). Additional work will be needed to determine if RBM26 influences the mRNA stability of DNA-PK components.  
  **Results:** mRNA stability;  chromatin remodeling
 
 - **Paper2:** [Xin, Yucui, and Yingsheng Zhang. "Paralog-based synthetic lethality: rationales and applications." Frontiers in Oncology 13 (2023): 1168143.](https://pmc.ncbi.nlm.nih.gov/articles/PMC10282757/). 
  **Species:** Human cell line  
  **Mechanism:** The RNA-binding proteins RBM26 and RBM27 play a critical role in mRNA processing, and their simultaneous depletion leads to a synergistic reduction in cell viability.   
  **Results:** cell viability  


- **Database:** [MGI-rbm26](https://www.informatics.jax.org/allele/MGI:5630069)
  **Species:**  Mice  
  **phenotype:** preweaning lethality, incomplete penetrance

## Rbm5  
 - **Paper1:** [XO'Bryan, Moira K., et al. "RBM5 is a male germ cell splicing factor and is required for spermatid differentiation and male fertility." PLoS genetics 9.7 (2013): e1003628.](https://pmc.ncbi.nlm.nih.gov/articles/PMC3723494/)  
  **Species:** Mice   
  **Mechanism:** The homozygous mutant Joey males (hereafter referred to as Rbm5sda/sda, where sda refers to the spermatid differentiation arrest phenotype) presented with male sterility at all ages examined (n = 30, age 8 weeks to 6 months old). Rbm5sda/sda mice were otherwise healthy as indicated by a body-wide pathology assessment (n = 10, 9 months old).     
  **Results:** male infertility

## Rbm10  
 - **Paper1:** [Rodor, Julie, et al. "The RNA-binding landscape of RBM10 and its role in alternative splicing regulation in models of mouse early development." RNA biology 14.1 (2017): 45-57.](https://www.tandfonline.com/doi/full/10.1080/15476286.2016.1247148)  
  **Mechanism:** Mutations in the RNA-binding protein, RBM10, result in a human syndromic form of cleft palate, termed TARP syndrome. A role for RBM10 in alternative splicing regulation has been previously demonstrated in human cell lines. To uncover the cellular functions of RBM10 in a cell line that is relevant to the phenotype observed in TARP syndrome, we used iCLIP to identify its endogenous RNA targets in a mouse embryonic mandibular cell line. We observed that RBM10 binds to pre-mRNAs with significant enrichment in intronic regions, in agreement with a role for this protein in pre-mRNA splicing. In addition to protein-coding transcripts, RBM10 also binds to a variety of cellular RNAs, including non-coding RNAs, such as spliceosomal small nuclear RNAs, U2 and U12. RNA-seq was used to investigate changes in gene expression and alternative splicing in RBM10 KO mouse mandibular cells and also in mouse ES cells. We uncovered a role for RBM10 in the regulation of alternative splicing of common transcripts in both cell lines but also identified cell-type specific events. Importantly, those pre-mRNAs that display changes in alternative splicing also contain RBM10 iCLIP tags, suggesting a direct role of RBM10 in these events. Finally, we show that depletion of RBM10 in mouse ES cells leads to proliferation defects and to gross alterations in their differentiation potential. These results demonstrate a role for RBM10 in the regulation of alternative splicing in two cell models of mouse early development and suggests that mutations in RBM10 could lead to splicing changes that affect normal palate development and cause human disease.     
  **Results:** proliferation defects, differentiation potential


- **Paper2:** [Gebauer, Fátima, et al. "RNA-binding proteins in human genetic disease." Nature Reviews Genetics 22.3 (2021): 185-198.](https://www.nature.com/articles/s41576-020-00302-y)
  **Species:**  Mice
  **Mechanism:** Mutations in the gene encoding the RBP RBM10 underlie the X-linked pleiotropic developmental TARP syndrome (OMIM #311900)79,80. RBM10 can function as a splicing regulator and, indeed, alterations in alternative splicing have been identified in lymphoblastoid cell lines derived from a patient with TARP syndrome, as well as in mandibular cells from Rbm10 knockout mice and mouse embryonic stem cells depleted of RBM10 (refs81,82). These findings suggest that splicing alterations are mechanistically involved in the pathogenesis of TARP syndrome. Possibly related to heart abnormalities in this syndrome, RBM10 can also regulate mRNA 3′-end formation of genes involved in cardiac development83. In addition to TARP syndrome, somatic mutations in the RBM10 gene occur in numerous cancers, including lung adenocarcinoma84,85,86,87, colorectal cancer88, pancreatic cancer89 and bladder cancer90. Consistent with context-specific functions of RBPs in cancer, RBM10 can function as a tumour suppressor or an oncogene91,92,93, possibly linked to expression of alternatively spliced variants and cross-regulatory feedback loops with the related protein RBM5 (ref.94). The functions of RBM10 as a tumour suppressor can, at least in part, be explained by modulation of alternative splicing of genes relevant for the control of cell growth and apoptosis, including the Notch regulator NUMB91,95,96. Mechanistically, RBM10 binds to specific sequence motifs in the pre-mRNA and inhibits inclusion of exons near RBM10 binding sites81,91,97, leading to defects on specific genes and to precise outcomes in the context of development and cancer.   
  **Results:**  cardiac development and cancer(lung adenocarcinoma, colorectal cancer, pancreatic cancer and bladder cancer).


## Rbm15  
 - **Paper1:**  [Hu, Liang, et al. "Loss of the RNA-binding protein Rbm15 disrupts liver maturation in zebrafish." Journal of Biological Chemistry 295.33 (2020): 11466-11472.](https://www.jbc.org/article/S0021-9258(17)48451-1/fulltext) 
  **Species:**  Zebrafish
  **Mechanism:**  Liver organogenesis begins with hepatic precursors in the foregut endoderm, followed by hepatoblast specification, differentiation, outgrowth, and maturation for the formation of functional hepatocytes. Although several signaling pathways and critical factors that regulate liver specification, differentiation, and proliferation have been identified, little is known about how liver maturation is regulated. Here, we used a screen for mutations affecting liver development in zebrafish and identified a cq96 mutant that exhibits a specific defect in liver maturation. Results from positional cloning revealed that cq96 encodes an RNA-binding protein, Rbm15, which is an evolutionarily conserved Spen family protein and known to play a crucial role in RNA m6A modification, nuclear export, and alternative splicing. However, a function of Rbm15 in embryonic liver development has not been reported. We found that Rbm15 is specifically expressed in the liver after its differentiation. CRISPR/Cas9-mediated loss of rbm15 repressed hepatic maturation, but did not affect hepatoblast specification, differentiation, and hepatocyte proliferation and apoptosis. Additional experiments disclosed that the mTOR complex 1 (mTORC1) pathway is highly activated in rbm15-deficient hepatocytes. Moreover, rapamycin treatment partially restored normal hepatic gene expression as well as the nuclear location of the transcription factor Hnf4a. Taken together, these results reveal an unexpected role of Rbm15 in liver maturation.    
  **Results:** liver maturation, Rbm15 mutant survived for a maximum of one month, and the mutants could not be detected in the adult fish.
- **Database:** [MGI-rbm15](https://www.informatics.jax.org/allele/MGI:3706995)
  **Species:**  Mice  
  **phenotype:** homozygous: embryonic lethality during organogenesis, complete penetrance

## Rbm28  
 - **Paper1:**  [Lin, Xin, et al. "RNA-binding protein RBM28 can translocate from the nucleolus to the nucleoplasm to inhibit the transcriptional activity of p53." Journal of Biological Chemistry 298.2 (2022).](https://www.jbc.org/article/S0021-9258(21)01334-X/fulltext)
  **Species:**  Mice  
  **Mechanism:** We further confirmed the effect of RBM28 on p53 using CDKN1A as the representative. According to dual luciferase reporter assay and chromatin immunoprecipitation (ChIP) assay, RBM28 depletion increased CDKN1A promoter activity and the binding of p53 to the CDKN1A promoter in U2OS cells (Fig. 3, E and F). More importantly, p21, which is the protein product encoded by CDKN1A was upregulated in p53 WT cells but not p53 null cells, suggesting that the effect of RBM28 on p53 target genes was p53-dependent (Fig. 3G). Collectively, these results suggested that RBM28 may impair the transcriptional activity of p53 by reducing the binding of p53 to its target gene promoters. In addition, reduction in the colony formation by depletion of RBM28 was weaker in HCT116 p53−/− cells than that in HCT116 p53+/+ cells, as shown in Fig. S1, indicating that RBM28 has other downstream effectors besides p53. This is consistent with the DepMap data results that RBM28 is required for cancer proliferation both in p53 WT and mutant cell lines, as shown in Fig. S2. Together, these results indicated that p53 is one of the downstream effectors of RBM28 to maintain cell proliferation.    
  **Results:**  transcriptional activity of p53, survival and growth of cancer cells

- **Database:** [MGI-rbm28](https://www.informatics.jax.org/allele/MGI:7612606)
  **Species:**  Mice  
  **mortality/aging:** preweaning lethality, complete penetrance


## Rbm39b
 - **Database:** [MGI-rbm39b](https://www.informatics.jax.org/allele/genoview/MGI%3A2679227?utm_source=chatgpt.com)
  **Species:**  Mice  
  **mortality/aging:** prenatal lethality, complete penetrance ( J:85162 )  


## Rbm19
- **Database:** [MGI-rbm19](https://www.informatics.jax.org/allele/MGI:3843656)
  **Species:**  Mice  
  **mortality/aging:** failure of morula compaction; embryonic growth arrest; embryonic lethality before implantation, complete penetrance
  

# HNRNP Family
![Hnrnp family](https://media.springernature.com/full/springer-static/image/art%3A10.1007%2Fs00439-016-1683-5/MediaObjects/439_2016_1683_Fig2_HTML.gif?as=webp)

## Hnrnpm
- **Paper1:** [Geuens, Thomas, Delphine Bouhy, and Vincent Timmerman. "The hnRNP family: insights into their role in health and disease." Human genetics 135.8 (2016): 851-867](https://link.springer.com/article/10.1007/s00439-016-1683-5)  
  **Species:** Mice  
  **Mechanism:** The hnRNP M is an abundant protein composed of three RRMs and is mostly present in the nucleus where it acts as a component of the spliceosome complex (Llères et al. 2010). This association is further confirmed by the identification of hnRNP M in nuclear speckles that are enriched with splicing factors (Marko et al. 2010). Since the first report where hnRNP M was identified as a splicing regulator for fibroblast growth factor receptor 2 (FGFR2) (Hovhannisyan and Carstens 2007), many reports followed and identified new targets: carcinoembryonic antigen-related cell adhesion molecule-1 (CEACAM1) (Dery et al. 2011), dopamine D2 receptor (D2M) (Park et al. 2011) and survival motor neuron 1 and 2 (SMN1 and SMN2) (Cho et al. 2014).

## Hnrnpk
- **Paper1:** [Geuens, Thomas, Delphine Bouhy, and Vincent Timmerman. "The hnRNP family: insights into their role in health and disease." Human genetics 135.8 (2016): 851-867](https://link.springer.com/article/10.1007/s00439-016-1683-5)  
  **Species:** Mice  
  **Mechanism:** There is strong evidence for hnRNP K to play an important role in the post-transcriptional regulation of multiple genes involved in the cytoskeletal organization of axons, and hnRNP K is therefore found to be essential for axonogenesis (Liu and Szaro 2011). This regulation is directed by the phosphorylation of hnRNP K by the c-Jun N-terminal kinase (JNK) (Hutchins and Szaro 2013). Corroborating this, the knockdown of hnRNP K results in the failure to create axons in developing Xenopus embryos (Liu et al. 2008). In addition, a mouse model harboring an hnRNP K knockout allele results in reduced survival and increased tumor formation (Gallardo et al. 2015). The importance of hnRNP K during embryonal development was supported by the appreciation of high protein expression levels of hnRNP K in the central and peripheral nervous system (Blanchette et al. 2006).  
  **Results:** axonogenesis of embryo; reduced survival  


# FUBP Family
## Fubp2/khsrp
- **Database:** [MGI-fubp2](https://www.informatics.jax.org/allele/MGI:5285092)  
  **Species:** Mice  
  **Phenotype:** Decreased susceptibility to Herpesvirales infection induced morbidity/mortality 
- **Paper1:** [CAI Renjie, XU Ming. KHSRP regulates the responsiveness of prostate cancer cells to androgens through ANK3[J]. Journal of Shanghai Jiao Tong University (Medical Science), 2024, 44(4): 417-426.](https://xuebao.shsmu.edu.cn/EN/Y2024/V44/I4/417) 
  **Species:** Mice
  **Mechanism:** GEO data analysis showed that KHSRP was upregulated in prostate cancer tissues compared to benign prostate tissues, suggesting its association with prostate tumorigenesis. Cell proliferation assay, colony formation assay, and mouse xenograft assay demonstrated a negative correlation between KHSRP expression and the proliferation ability of LNCaP cells, indicating that KHSRP can inhibit the proliferation of prostate cancer cells, with a stronger effect on LNCaP cells than on DU145 cells. Western blotting and RT-qPCR analysis of the stable LNCaP cell lines showed that KHSRP overexpression led to a decrease in AR protein level without affecting its mRNA level, suggesting that KHSRP can indirectly regulate AR protein level. RNA-seq and RT-qPCR results showed KHSRP was positively correlated with the expression of ANK3, a regulatory factor affecting AR protein stability, and subsequent Western blotting confirmed an increase in ANK3 protein expression after KHSRP overexpression. TCGA data analysis further supported the correlation between KHSRP and ANK3 mRNA expression. Interestingly, according to CCLE and GEO data, the expression of ANK3 was closely related to prostate cancer malignancy. 
  **Result:** KHSRP may indirectly regulate AR protein stability through ANK3, thereby influencing the proliferation of androgen-dependent prostate cancer cells and mediating the altered responsiveness to androgen in prostate cancer cells.

## Fubp1
- **Paper1:** [Zhou, Weixin, et al. "Far upstream element binding protein plays a crucial role in embryonic development, hematopoiesis, and stabilizing Myc expression levels." The American journal of pathology 186.3 (2016): 701-715.](https://www.sciencedirect.com/science/article/pii/S0002944015006562)
  **Species:** Mice  
  **Mechanism:** The transcription factor far upstream element binding protein (FBP) binds and activates the MYC promoter when far upstream element is via TFIIH helicase activity early in the transcription cycle. The fundamental biology and pathology of FBP are complex. In some tumors FBP seems pro-oncogenic, whereas in others it is a tumor suppressor. We generated an FBP knockout (Fubp1−/−) mouse to study FBP deficiency. FBP is embryo lethal from embryonic day 10.5 to birth.  
  **Result**:  Embryonic lethality  
- **Database:** [MGI-fubp1](https://www.informatics.jax.org/allele/MGI:5789037)  
  **Species:** Mice  
  **Phenotype:**  Embryonic lethality  

## Fubp3
- **Paper3:** [Izsvák, Zsuzsanna, et al. "Generating knockout rats by transposon mutagenesis in spermatogonial stem cells." Nature methods 7.6 (2010): 443-445.](https://www.nature.com/articles/nmeth.1461#MOESM294)
  **Species:** Mice
  **Mechanism:** By this approach, we generated 35 rat lines with defined gene-trap mutations, in which greater than 50% of the trapped genes (Abca13, Alms1, Arntl, Btrc, RGD1560888 (similar to mouse Cdk8), Cdh17, Dlg1, Exoc6b, Fubp3, Grik3, RGD1562674 (similar to mouse Ksr2), Pclo, Zmynd8 (also known as Prkcbp1), RGD1561493 (similar to mouse Rgs22), Ssbp2, Slc35a3, Tmem57, Tbc1d1 and Ube2k) were associated with biological processes linked to an assortment of human diseases (Supplementary Table 3).
  **Result:** Proto-oncogene; Viral replication
 


# Others
## Raver2
- **Database:** [MGI-raver2](https://www.informatics.jax.org/allele/MGI:6294074)
  **Species:**  Mice  
  **phenotype:** behavior/neurological: decreased grip strength; homeostasis/metabolism: decreased circulating phosphate level

## Puf60a
- **Paper1:** [Zhang, Yue, et al. "Genome-wide CRISPR screen identifies Puf60 as a novel Stemness gene of mouse embryonic stem cells." Stem Cells and Development 31.5-6 (2022): 132-142.](https://www.liebertpub.com/doi/abs/10.1089/scd.2021.0309)
 **Species:**  Mice  
  **Mechanism:** The mechanisms underlying self-renewal of embryonic stem cells (ESCs) hold great value in the clinical translation of stem cell biology and regenerative medicine research. To study the mechanisms in ESC self-renewal, screening and identification of key genes maintaining ESC self-renewal were performed by a genome-wide CRISPR-Cas9 knockout virus library. The mouse ESC R1 were infected with CRISPR-Cas9 knockout virus library and cultured for 14 days. The variation of single guide RNA (sgRNA) ratio was analyzed by high-throughput sequencing, followed by bioinformatics analysis to profile the altered genes. Our results showed 1375 genes with increased sgRNA ratio were found to be mainly involved in signal transduction, cell differentiation, and cell apoptosis; 2929 genes with decreased sgRNA ratio were mainly involved in cell cycle regulation, RNA splicing, and biological metabolic processes. We further confirmed our screen specificity by identifying Puf60, U2af2, Wdr75, and Usp16 as novel positive regulators in mESC self-renewal. Meanwhile, further analysis showed the relevance between Puf60 expression and tumor. In conclusion, our study screened key genes maintaining ESC self-renewal and successfully identified Puf60, U2af2, Wdr75, and Usp16 as novel positive regulators in mESC self-renewal, which provided theoretical basis and research clues for a better understanding of ESC self-renewal regulation.
  **Results:** tumor

  
## Pcbp2
- **Database：** [MGI-pcbp2](https://www.informatics.jax.org/allele/MGI:5908914)
  **Species:**  Mice  
  **phenotype:** homozygous: lethality throughout fetal growth and development, complete penetrance; postnatal lethality, complete penetrance;embryonic lethality during organogenesis, incomplete penetrance


## si:dkeyp-121d4.3
- **Database:** [ZFIN-si:dkeyp-121d4.3](https://zfin.org/ZDB-GENE-130530-962#phenotype)
  **Species:** Zebrafish  
  **Phenotype:** Premature Stop

## Zfr
- **Database:** [MGI-zfr](https://www.informatics.jax.org/allele/MGI:2387996)  
  **Species:** Mice  
  **Phenotype:** homozygous: embryonic lethality between somite formation and embryo turning, complete penetrance


## Unk
- **Database:** [MGI-unk](https://www.informatics.jax.org/allele/MGI:4880910)  
  **Species:** Mice  
  **Phenotype:** preweaning lethality, incomplete penetrance  


## Celf1
- **Database:** [MGI-celf1](https://www.informatics.jax.org/allele/MGI:3700209)  
  **Species:** Mice  
  **Phenotype:** postnatal lethality, incomplete penetrance

## Bicc1a
- **Database:** [MGI-bicc1](https://www.informatics.jax.org/allele/MGI:6276939)  
  **Species:** Mice  
  **Phenotype:** preweaning lethality, incomplete penetrance



